tests/test_data/str-axes.json

```diff
@@ -0,0 +1,8 @@
+{
+    "x": {
+        "values": [
+            "1",
+            "2"
```

Comment:
This doesn't look "happy" to me. I don't think the intention of the spec is that numbers can be given as string, but instead that the axis something like "red", "green", "blue".

Reply:
Agreed. Change to 'real' strings.